### PR TITLE
HTML/CSS & JavaScript capstone project

### PR DESCRIPTION
**Pull request by carlosmuerto**
# Project requirements

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,7 +15,218 @@
...
+        </div>
+
+      </div>
+
+      <div class="more-btn">
+        <button type="button">
+          I wanna see more persons
+          <i class="fa-solid fa-chevron-down more-btn-chevron-down"></i>
+        </button>
+      </div>
```
Comment (keroloussamy) [inline index.html:175]: - [x] Kindly make sure the all cards are aligned with each other.

image

File: index.html [hunk 2]
```diff
@@ -13,7 +15,218 @@
...
+            <span>Meaning:</span> A pair of people who have a shared love for each other.
+          </p>
+        </div>
+      </div>
+
+      <div class="categories-btn">
+        <button type="button">See More</button>
+      </div>
+
+    </section>
```
Comment (keroloussamy) [inline index.html:143]: - [x] Kindly change the style for this button, to match the original design.

Your design
image

Original design 
image

File: index.html [hunk 3]
```diff
@@ -13,7 +15,218 @@
...
 <body>
-  <h1>Conference Page</h1><i class="fa-solid fa-computer"></i>
+  <div class="wrapper social-top">
+    <ul>
+      <li>
+        <a class="icon" href="https://github.com/carlosmuerto/capstone-project-conference-page">
+          <i class="fa-brands fa-github"></i>
+        </a>
+      </li>
+    </ul>
```
Comment (keroloussamy) [inline index.html]: - [x] Kindly add the rest of links to match the original layout.

image

image

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @carlosmuerto, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._

- [x] Kindly add the link for a video presenting the features of the project, in your README file.
- [x] Kindly update your link for your deployed project, in your README file.



## Optional suggestions
_Check the comments under the review._

- [x] x

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MeqdamAlqudah — APPROVED** [review]
Hi @carlosmuerto , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**